docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Building for TV
```

Comment:
```suggestion
title: Build apps for TV
```

I think we can add more clarity to the title by adding the words "apps" and also use short form verb for page titles instead of gerunds.